static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -380,6 +389,15 @@ class RuleConditionsForm extends PureComponent<Props, State> {
     return (
       <Fragment>
         <ChartPanel>
+          {isExtrapolatedChartData && (
+            <OnDemandMetricInfoAlert>
+              <IconInfo size="sm" color="blue400" style={{marginRight: '10px'}} />
```

Comment:
can we use emotion to style this icon? :)

Reply:
sure!